test/data/command.update.yaml

```diff
@@ -0,0 +1,4 @@
+# Command UPDATE example
```

Comment:
It might be better to comment that UPDATE or DELETE is not scope at this chance.

Reply:
I added this PR's goal (MVP) to my first comment.
Thanks.

Reply:
I've checked and understood. Thank you.

Reply:
We can probably remove this file